examples/nextjs-page-router/pages/index.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from '@/styles/Home.module.css';
+import { Inter } from 'next/font/google';
+
+import dynamic from 'next/dynamic';
+
+const DynamicWidget = dynamic(() => import('@/components/Widget'), {
+  loading: () => <p>Loading...</p>,
```

Comment:
Can we use our Skeleton here?

Reply:
We need to do a it of work to allow that. Looking at it now